packages/components/stories/List.js

```diff
@@ -432,6 +450,31 @@ storiesOf('List', module)
 			</div>
 		);
 	})
+	.add('Virtualized - row click', () => {
```

Comment:
Change it to `Virtualized - activation` to stay with the same terminology as `Virtualized - selection`
Selection covers onToggle + isSelected
Activation covers onRowClick + isActive